elements/pfe-sass/mixins/_mixins.scss

```diff
@@ -192,6 +193,29 @@
   }
 }
 
+/// ===========================================================================
+/// Tab-specific SASS mixins
+/// ===========================================================================
+
+@mixin pfe-tab-panel-container($orientation: "horizontal", $border: false) {
```

Comment:
In the meantime, until we know if there's a reuse for this mixin, should we move this into the pfe-tab-panel sass file at the top?  What do you think @starryeyez024 ?